### DISCUSSION TITLE
eukCC in metaGEM

### DISCUSSION THREAD

**Discussion by ycsong** [franciscozorrilla/thesis_pipeline#159] [General]
Hi Francisco,

I have been running metaGEM successfully from raw sequence to GTDBtk results for a while now. I had to revisit the metaGEM paper for writing my own manuscript, and noticed in Figure 1. that EukCC is also available as part of the workflow. Our group is interested in this, as we are currently exploring methods of detecting/annotating fungal genomes in our data.

However it looks like EukCC is currently not implemented in metaGEM, as I wasn't able to invoke the function using '-t eukcc'. I also checked the Snakemake file to see if this has been implemented there, but doesn't appear so. Is there some sort of set-up/installation procedures that need to be done beforehand?

Thank you for your time,

Young

**Comment by franciscozorrilla** [comment]
Hi Young,

Thanks for pointing this out, you are correct eukcc is not fully implemented in the Snakefile/metaGEM wrapper. Not sure if the code got lost somewhere along the way or I never got around to implementing it since the results of the paper focused only on the prokaryotic genomes and corresponding metabolic models, either way I will create a PR/issue to address this (#160). 

I had a quick look at the eukcc [repo](https://github.com/EBI-Metagenomics/EukCC) and it seems like there have been significant changes since the previous version I was using, so the old code is likely not useful for you. For expediency, I would recommend that you use eukcc as a standalone tool, it looks like there is good [documentation](https://eukcc.readthedocs.io/en/latest/) for setup and usage.

Good luck!
Francisco

**Reply by ycsong** [reply]
many thanks for the prompt response, Francisco. Will certainly give the stand-alone version a go.
